tests/Behat/Context/Application/CreditMemoContext.php

```diff
@@ -109,6 +110,32 @@ public function creditMemoTotalShouldBe(int $total): void
         Assert::same($this->creditMemo->getTotal(), $total);
     }
 
+    /**
+     * @Then /^its subtotal should be ("[^"]+")$/
+     */
+    public function creditMemoSubtotalShouldBe(int $subtotal): void
+    {
+        Assert::same($this->creditMemo->getSubtotal(), $subtotal);
+    }
+
+    /**
+     * @Then /^it should have a tax item "([^"]+)" with amount ("[^"]+")$/
+     */
+    public function itShouldHaveATaxItemWithAmount(string $label, int $amount): void
+    {
+        $result = false;
+        /** @var TaxItemInterface $taxItem */
+        foreach ($this->creditMemo->getTaxItems() as $taxItem) {
+            if ($taxItem->getLabel() === $label && $taxItem->getAmount() === $amount) {
+                $result = true;
+
+                break;
```

Comment:
We could just make a `return` there and throw an exception in line 136 (if it reaches this line it means a tax item with such a label and amount has not been found :dancer:)